recipe/migrations/pypy.yaml

```diff
@@ -0,0 +1,19 @@
+migrator_ts: 1580746218   # The timestamp of when the migration was made
+__migrator: 
+  kind: 
+    version
+  exclude:
+    - c_compiler
+    - vc
+    - cxx_compiler
+  migration_number:  # Only use this if the bot messes up, putting this in will cause a complete rerun of the migration
+    1 
+  bump_number: 0
+
+python:
```

Comment:
Does this need a modification to the zip keys?

Reply:
Nope. Not building for windows here.

Reply:
Ahh ok